cluster/log-dump/log-dump.sh

```diff
@@ -229,31 +229,23 @@ function save-logs-windows() {
       echo "Not saving logs for ${node}, Windows log dumping requires gcloud support"
       return
     fi
-
-    export-windows-docker-event-log "${node}"
```

Comment:
No, we cannot call it, as it runs powershell cmd over SSH, and I left the function definition there in case we change back to SSH approach later. Diagnostics cmd collected all the event logs including docker event in evtx files, but not an explicit docker.log file. We could do the same in diagnostics cmd, if needed.

Reply:
How can one read the evtx files? Is it possible to open them on a Linux workstation?

If we need to transform them somehow perhaps we should do this after unzipping the archive file at L246.

Reply:
Right, there're some tools like [python-evtx](https://github.com/williballenthin/python-evtx), [evtViewer](https://sourceforge.net/projects/evtviewer/) etc. But still not user-friendly, looks we need to improve diagnostics to dump plain text logs.

Reply:
Ok. If there's something easy to do then feel free to add it to this PR, otherwise we can follow-up with changes here or in the diagnostics tool itself.

Reply:
Tried using python-evtx tool on sample evtx file, looks like it converts evtx to xml, still not human-readable. evtViewer seems not working. Thinking to change it in the diagnostics tool.

Reply:
Yeah, that sounds like probably the best approach. Consider filing a FR issue in https://github.com/GoogleCloudPlatform/compute-image-tools/ to track this.

Reply:
Done thru PR: https://github.com/GoogleCloudPlatform/compute-image-tools/pull/979